src/DBImport.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace TheRealMVP;
+
+class DBImport
+{
+    protected $pdo;
+
+    /**
+     * DBImport constructor.
+     *
+     * Creates new PDO object
+     */
+    public function __construct()
+    {
+        $this->pdo = new \PDO ("mysql:host=db; dbname=TheRealMVP", "root", "password");
+    }
+
+    /**
+     * Drops all tables and recreates them
+     *
+     * Populates all tables
+     *
+     * @param $apiData
+     *                Array of data from api
+     */
+    public function storeData(array $apiData): void
```

Comment:
This method does quite a bit itself, it could grow to be quite large if the API functionality was expanded.

At a minimum, I would suggest moving the functionality to drop and re-create the tables into a separate method.